pkgs/os-specific/bsd/netbsd/default.nix

```diff
@@ -204,9 +202,15 @@ in lib.makeScopeWithSplicing
       "TSORT=cat"
       # Can't process man pages yet
       "MKSHARE=no"
+    ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      # GNU objcopy produces broken .a libs which won't link into dependers.
+      # Makefiles only invoke `$OBJCOPY -x/-X`, so cctools strip works here.
+      "OBJCOPY=${buildPackages.darwin.cctools}/bin/strip"
```

Comment:
Oops: I meant to ask why we're explicitly referencing cctools here: `strip` is already on the PATH, so we shouldn't need to (and we're expecting to use the same toolchain for this as everything else; since we use strip all over the place in nixpkgs if it can't deal with Mach-O binaries you're already in a World of Pain)

Reply:
Yeah that's fair. Just wanted to make it abundantly clear that you have to use the special cctools strip on macOS. I guess that's already implied by $STRIP when hostPlatform.isDarwin.